deps/bindeps.jl

```diff
@@ -181,7 +181,7 @@ function bindeps_setup()
         :device_libs_build_reason => "unknown",
     )
     for (name, _) in rocm_ext_libs
-        lib = Symbol(:lib, lowercase(string(name)))
+        lib = Symbol(:lib, name == :MIOpen ? string(name) : lowercase(string(name)))
```

Comment:
```suggestion
        lib = Symbol(:lib, name)
```

Since the others are already lowercase. I think the `lowercase` call was just a relic of older code.